src/components/homePageMainContent/homePageMainContent.tsx

```diff
@@ -1,13 +1,36 @@
 import React from "react";
 import FeaturedContentComponent from "../featuredContent/featuredContent";
 import SwimLanes from "../swimlanes/swimLanes";
+import { useEffect, useState } from "react";
+import SwimLanesLoading from "../swimlanes/swimLanesLoading";
 
-const HomePageMainContent = ({ randomNumber, lanesWithNumItems }) => {
-  return (
+const HomePageMainContent = () => {
+  // const [isSwimLaneLoading, setIsSwimLaneLoading] = useState(false);
+  const [data, setData] = useState<any>({});
```

Comment:
Similar idea here to the featured item check, it's okay to use the availability of `data` and its properties for the loading state.